FinniversKit/Sources/DNA/Color/ColorProvider.swift

```diff
@@ -129,13 +189,13 @@ public struct DefaultColorProvider: ColorProvider {
     public var textDisabled: UIColor {
         .dynamicColor(defaultColor: .blueGray300, darkModeColor: .darkSardine)
     }
-    
+
     public var textAlert: UIColor {
         .lightNuttyBrown
     }
 
     public var textCritical: UIColor {
-        .dynamicColor(defaultColor: .red600, darkModeColor: .red400)
+        .dynamicColor( defaultColor: .red600, darkModeColor: .red400 )
```

Comment:
can remove spaces here for consisteny 😄

Reply:
Haha...both @BjornRuud and you found this 🔍  

fixed ✅